Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -241,74 +234,55 @@ public void onResult(@Nullable List<PaymentMethodNonce> vaultedPaymentMethods, @
         }
     }
 
-    private void onDismissVaultManager(DropInEvent event) {
-        showSelectPaymentMethodFragment();
+    private void onAddCardSubmit(DropInEvent event) {
+        String cardNumber = event.getString(DropInEventProperty.CARD_NUMBER);
+        showCardDetailsFragment(cardNumber);
     }
 
-    private void showSelectPaymentMethodFragment() {
+    private boolean shouldAddFragment(String tag) {
         FragmentManager fragmentManager = getSupportFragmentManager();
-        Fragment fragment = fragmentManager.findFragmentByTag("SELECT_PAYMENT_METHOD");
-        if (fragment == null) {
-            Bundle args = new Bundle();
-            args.putParcelable("EXTRA_DROP_IN_REQUEST", mDropInRequest);
+        Fragment fragment = fragmentManager.findFragmentByTag(tag);
+        return (fragment == null);
+    }
 
-            fragmentManager
-                    .beginTransaction()
-                    .setReorderingAllowed(true)
-                    .replace(R.id.fragment_container_view, SelectPaymentMethodFragment.class, args, "SELECT_PAYMENT_METHOD")
-                    .commit();
-        }
+    private void replaceExistingFragment(Class <? extends Fragment> fragmentClass, String tag, Bundle args) {
+        FragmentManager fragmentManager = getSupportFragmentManager();
+        fragmentManager
+                .beginTransaction()
```

Comment:
Looks like we lost the `setReorderingAllowed(true)` here. I don't know that we need it, but wanted to call that out.

Reply:
Yeah it didn't seem like it was needed. It has more to do with animations / transitions. Wanted to save those for last to clear the way for us to reach feature parity without thinking about animating.